src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -2082,6 +2087,12 @@ private val BlockingReason.obstructionFreedomViolationMessage: String get() = wh
     is BlockingReason.Suspended    -> OBSTRUCTION_FREEDOM_SUSPEND_VIOLATION_MESSAGE
 }
 
+enum class ExecutionMode {
+    REGULAR,
+    GENERAL_PURPOSE_MODEL_CHECKER,
```

Comment:
GPMC?

Reply:
The short version may not be obvious enough when you first time see GPMC. I would leave it as is